spirv_msl.cpp

```diff
@@ -2917,7 +2917,14 @@ void CompilerMSL::emit_sampled_image_op(uint32_t result_type, uint32_t result_id
 // Manufacture automatic sampler arg for SampledImage texture.
 string CompilerMSL::to_func_call_arg(uint32_t id)
 {
-	string arg_str = CompilerGLSL::to_func_call_arg(id);
+	// Make sure that we use the name of the original variable, and not the parameter alias.
```

Comment:
This is so close to CompilerGLSL::to_func_call_arg that not being able to use it feels wrong.